compute/functions/reference-content/code-examples.mdx

```diff
@@ -195,6 +195,9 @@ Please refer to our guide on [code packaging](https://www.scaleway.com/en/docs/c
 
 ### Returning a function result in Node
 
+**Note** Following examples uses `CommonJS` way for modules, if you consider using ES6 Modules 
```

Comment:
<Message type="note">

The following examples uses `CommonJS` modules. If you use ES6 Modules refer to the end of this section.

</Message>